round4/TabTracker.md

```diff
@@ -0,0 +1,109 @@
+3. As a user who often multitasks, I want to easily spot tabs that have been open for a long time so that I can prioritize which tabs to focus on.
+4. As a developer, I want to manage tabs related to different coding projects and documentation to enhance my productivity.
+5. As a project manager, I want to easily spot and close tabs related to completed or outdated projects to declutter my browser.
+6. As a user, I can easily manage my tabs to avoid having tabs of the same information.
+7. As a user, I want to be able to set a timer threshold such that TabTracker will close tabs which pass this threshold.
+8. As a student, I want to clearly see which of my tabs are inactive and which ones are active
+9. As a student, I want to easily be able to close multiple inactive tabs at once instead of individually
+10. As a user I want a scroll feature that allows me to easily see all of the tabs I have open
+11. As a teacher with many papers to review online, TabTracker allows me to reduce clutter on my browser by prompting me to close those papers I’ve already graded.
+12. As the CEO of Google, I am tired of Chrome not doing anything to help fix my excessive tab issue and can now have an efficient browsing experience.
+13. As a parent, I can get a prompt to close all the tabs my kid has opened when he is finished using my computer to play video games.
+14. As a student with many classes, I want my tabs in different browsers to show up on a holistic breakdown.
+15. As a student athlete, I can more efficiently get schoolwork done by closing unnecessary outdated sports-related tabs.
+
+## Notes
+
+Certain potential features:
+Ordering tabs by most recently used
+Tab grouping with color-coded columns
+Implementing a search feature
+
+## References & Inspiration
+
+OneTab: <https://chromewebstore.google.com/detail/onetab/chphlpgkkbolifaimnlloiipkdnihall?pli=1>
+
+SessionBuddy -
+<https://chromewebstore.google.com/detail/session-buddy/edacconmaakjimmfgnblocblbcdcpbko>
+
+## Technical Details
+
+### User Interface
+
+Upon activating the extension, a side panel emerges displaying the activity and inactivity of each tab. The names and contents of each tab will be arrayed vertically and ordered by most recently used. The inactive tabs will be marked by red color-coding, whereas the active tabs will be marked by green color-coding. A prompt will also emerge to close all those highlighted red tabs, with an option to deselect some or all tabs to escape deletion. Additionally, hovering over the tabs in the sidebar will allow you to directly access any said tab through clicking on it.
+
+TabTracker implements a sidebar to display the names and contents of various tabs open and is navigable via a scrolling feature. Each tab will take up one line (two sections: website name and one-sentence summary of page content).
```

Comment:
I think this is smart. Something to think about is how you will consider what is active and what is inactive. For example, will you have a timer that if the tab has not been open within x amount of minutes/hours/days it will be considered inactive